Buildings/Applications/DataCenters/ChillerCooled/Examples/NonIntegratedPrimarySecondaryEconomizer.mo

```diff
@@ -2,41 +2,51 @@ within Buildings.Applications.DataCenters.ChillerCooled.Examples;
 model NonIntegratedPrimarySecondaryEconomizer
   "Example that demonstrates a chiller plant with non-integrated primary-secondary side economizer"
   extends Modelica.Icons.Example;
-
-  extends
-    Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PartialDataCenter(
+  extends Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PostProcess(
+    freCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.FreeCooling)
+      then 1 else 0),
+    parMecCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.PartialMechanical)
+      then 1 else 0),
+    fulMecCooSig(
+      y=if cooModCon.y == integer(Buildings.Applications.DataCenters.Types.CoolingModes.FullMechanical)
+      then 1 else 0),
+    PHVAC(y=cooTow[1].PFan + cooTow[2].PFan + pumCW[1].P + pumCW[2].P + sum(
+          chiWSE.powChi) + sum(priPum.P) + sum(secPum.P) + ahu.PFan + ahu.PHea),
+    PIT(y=roo.QSou.Q_flow));
+  extends Buildings.Applications.DataCenters.ChillerCooled.Examples.BaseClasses.PartialDataCenter(
     redeclare Buildings.Applications.DataCenters.ChillerCooled.Equipment.NonIntegrated chiWSE(
       controllerType=Modelica.Blocks.Types.SimpleController.PI,
       Ti=60,
       energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
       use_controller=false),
-    ahu(energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial, dp1_nominal=
-          60000),
-    pumCW(each energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial),
-    dpSet(k=80000),
-    pumSpe(k=1),
-    chiStaCon(dT=0.5,
-      tWai=0,
-      criPoiTem=553.86),
-    CWPumCon(tWai=0));
+    ahu(energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+      dp1_nominal = 60000),
+    pumCW(each energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial, each
+        use_inputFilter=false),
```

Comment:
Set these parameters correctly in the base class rather than in the redeclaration